modules/end-user-guide/examples/proc_using-the-che-prod-editor-file-to-select-an-ide.adoc

```diff
@@ -0,0 +1,18 @@
+[id="using-the-che-prod-editor-file-to-select-an-ide_{context}"]
+== Using the {prod-short} editor file to select an IDE
+
+Use the `che-editor.yaml` file to define a default IDE for the project users. For a list of supported IDs, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[Optional parameters for the URLs for starting a new workspace].
+
+.Procedure
+
+. Place the `che-editor.yaml` file in the `.che` folder in the root directory of your project. 
+. In the `che-editor.yaml` file, specify the ID of the IDE you are selecting. For example:
++
+====
+id: che-incubator/che-idea/latest
+====
```

Comment:
```suggestion
[source,yaml]
----
id: che-incubator/che-idea/latest
----
```

Reply:
You don't need to do so here, because you've said `For example:`. However, it is common, at least in the OpenShift docs, to label larger code blocks that contain example values as follows:
```
.Example
[source,yaml]
----
big custom resource definition file example
----
```